1-js/02-first-steps/04-variables/1-hello-variables/solution.md

```diff
@@ -9,4 +9,5 @@ admin = name;
 
 alert( admin ); // "John"
 ```
+```
```

Comment:
s-a adăugat o linie goală fără să inlocuim cu altceva. Incercam să nu schimbam ordinea liniilor, le lăsăm "așa cum sunt" si tot ce tine de formatare in general. 😁